user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md

```diff
@@ -59,19 +69,20 @@ For Cassandra, any number of nodes can be used, ideally running on Linux machine
 
 ### Failover setups (without geo-redundancy)
 
-A Failover setup is similar to the previously mentioned setups. You can consider the Failover pair to be like one DMA in the DMS. Both DataMiner instances point to the same Cassandra and Elasticsearch cluster. With this setup, you both get the DataMiner resilience and resilience in your Cassandra and Elasticsearch cluster.
+A Failover setup is similar to the previously mentioned setups. You can consider the Failover pair to be like one DMA in the DMS. Both DataMiner instances point to the same databases. For added resilience, you can use **managed services in the cloud** instead of a Cassandra cluster and Elasticsearch/OpenSearch cluster on premises. At present, we support the Amazon Keyspaces Service and Amazon OpenSearch Service.
 
-As an absolute minimum, such a system could consist of two DataMiner machines, one Elasticsearch machine and one Cassandra machine. However, if the Cassandra or Elasticsearch machine are no longer available in this setup, you will no longer be able to start DataMiner, so this is not recommended for redundancy setups.
-
-To be fully hardware-redundant, we recommend at least three machines hosting Elasticsearch nodes, two machines hosting the DataMiner Failover pair and two machines hosting Cassandra nodes. More Cassandra and Elasticsearch nodes can be needed depending on the load on your system. Check the [node calculator](https://community.dataminer.services/calculator/) for more information.
+An on-premises setup should at the very least consist of two DataMiner machines, one Elasticsearch machine, and one Cassandra machine. However, if the Cassandra or Elasticsearch machine are no longer available in this setup, you will no longer be able to start DataMiner, so this is not recommended for redundancy setups. To be fully hardware-redundant, we recommend **at least three machines hosting Elasticsearch** nodes, **two machines hosting the DataMiner Failover pair** and **two machines hosting Cassandra nodes**. More Cassandra and Elasticsearch nodes can be needed depending on the load on your system. Check the [**node calculator**](https://community.dataminer.services/calculator/) for more information.
```

Comment:
Note that the node calculator page says its under development and is a preview but also says the latest updated was the first of april last year. Is this up to date?

Reply:
@BertVandenberghe, what do you think? Can we refer to the node calculator? Does that page still need the preview label?

Reply:
Yes, you can definitely refer to the node calculator. As things are still evolving and being evaluated, the preview label will still remain for the time being.